spec/amp-actions-and-events.md

```diff
@@ -71,84 +71,123 @@ For example, the following is possible in AMP.
 ```
 
 ## Element Specific Events
+### form
```

Comment:
It seems good.  I would recommend putting the elements in alphabetical order (just move sidebar before user-notification).